### PR TITLE
Chore: Update Reanimated to 2.2.2

### PR DESCRIPTION
**Pull request by gerzonc**
## How to test or reproduce

- Test long pressing the message to open action sheet
- Test Image zoom on Android

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by AlexAlexandre — CHANGES_REQUESTED** [review]
The code it's ok, but could you please merge the latest changes from develop into this branch ?

**Review by AlexAlexandre — APPROVED** [review]
LGTM

**Review by diegolmello — COMMENTED** [review]
@gerzonc don't we need these extra steps on Android? https://docs.swmansion.com/react-native-reanimated/docs/installation#android

**Comment by gerzonc** [comment]
> @gerzonc don't we need these extra steps on Android? https://docs.swmansion.com/react-native-reanimated/docs/installation#android

It seems it was working fine without it, but I added it anyway just in case something didn't get to work fine with further changes we might do.

File: app/containers/ActionSheet/ActionSheet.tsx [hunk 1]
```diff
@@ -132,11 +132,12 @@ const ActionSheet = React.memo(
 		const renderItem = ({ item }: any) => <Item item={item} hide={hide} theme={theme} />;
 
 		const animatedPosition = React.useRef(new Value(0));
-		const opacity = interpolate(animatedPosition.current, {
+		// TODO: Similar to https://github.com/wcandillon/react-native-redash/issues/307#issuecomment-827442320
+		const opacity = interpolateNode(animatedPosition.current, {
```
Comment (diegolmello) [inline app/containers/ActionSheet/ActionSheet.tsx:136]: https://docs.swmansion.com/react-native-reanimated/docs/2.2.0/migration#1-interpolate-renamed-to-interpolatenode

File: app/presentation/ImageViewer/ImageViewer.android.tsx [hunk 2]
```diff
@@ -94,7 +94,7 @@ function runTiming(clock: any, value: any, dest: any, startStopClock: any = true
 	const config = {
 		toValue: new Value(0),
 		duration: 300,
-		easing: Easing.inOut(Easing.cubic)
+		easing: EasingNode.inOut(EasingNode.cubic)
```
Comment (diegolmello) [inline app/presentation/ImageViewer/ImageViewer.android.tsx:97]: https://docs.swmansion.com/react-native-reanimated/docs/2.2.0/migration#2-easing-renamed-to-easingnode

**Pull request merged**